app/src/Command/ReportSendMailCommand.php

```diff
@@ -92,8 +89,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
                 $tableMsgs = $this->twig->render('report/table_mail_msgs.html.twig', [
                     'untreatedMsgs' => $untreatedMsgs,
-                    'url' => $url,
-                    'user' => $user,
+                    'token' => $token,
```

Comment:
Tu peux virer cette ligne et garder la ligne "user" (mauvaise résolution de conflit Git je pense) :


```suggestion
                    'user' => $user,
```